python/tests/io/test_pyarrow.py

```diff
@@ -1345,3 +1374,655 @@ def test_pyarrow_wrap_fsspec(example_task: FileScanTask, table_schema_simple: Sc
 bar: [[1,2,3]]
 baz: [[true,false,null]]"""
     )
+
+
+def construct_test_table() -> Tuple[Any, Any, Union[TableMetadataV1, TableMetadataV2]]:
```

Comment:
I think the tests for Parquet stats should go into a separate test file. There's too much in `test_pyarrow` as it is and this is a reasonably self-contained set of tests.

Reply:
`test_pyarrow_stats.py`?